maintainers/maintainer-list.nix

```diff
@@ -423,6 +423,12 @@
     github = "abysssol";
     githubId = 76763323;
   };
+  a-camarillo = {
```

Comment:
from the github checks that ran:

 trace: maintainer acairncross (line 432) should be listed after abysssol, not after a-camarillo (line 426)